modules/ding_frontend/ding_frontend.strongarm.inc

```diff
@@ -91,7 +91,7 @@ background-attachment: fixed;',
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'languageicons_path';
-  $strongarm->value = 'profiles/ding2/modules/contrib/languageicons/flags/*.png';
```

Comment:
This should not be changed for all clients, this is custom for this client.
Can it be set with `drush vset`?